packages/x-charts-pro/src/Heatmap/HeatmapItem.tsx

```diff
@@ -95,7 +95,7 @@ function HeatmapItem(props: HeatmapItemProps) {
   const Cell = slots?.cell ?? HeatmapCell;
   const cellProps = useSlotProps({
     elementType: Cell,
-    additionalProps: { ...getInteractionItemProps({ type: 'heatmap', seriesId, dataIndex }) },
+    additionalProps: { ...interactionProps },
```

Comment:
Why not `additionalProps: interactionProps,`?

Reply:
Types were complaining, fixed it by adding proper return types to the `useInteractionItemProps`